src/ariel-os-rp/src/uart.rs

```diff
@@ -0,0 +1,259 @@
+//! UART configuration.
+
+#![expect(unsafe_code)]
+
+use ariel_os_embassy_common::{impl_async_uart_for_driver_enum, uart::ConfigError};
+
+use embassy_rp::{
+    Peripheral, bind_interrupts, peripherals,
+    uart::{BufferedInterruptHandler, BufferedUart, RxPin, TxPin},
+};
+
+/// UART interface configuration.
+#[derive(Debug, Clone, PartialEq, Eq)]
```

Comment:
I think this [should be `Copy` as well](https://doc.rust-lang.org/stable/std/marker/trait.Copy.html#when-should-my-type-be-copy):

```suggestion
#[derive(Debug, Copy, Clone, PartialEq, Eq)]
```

Reply:
Yup, missed that one somehow. Added in e55744ca